docs/changelog.rst

```diff
@@ -6,6 +6,8 @@ Changelog
 
 New features:
 
+* :doc:`plugins/keyfinder`: Added support for `keyfindercli`_
```

Comment:
I know this may seem overly pedantic, but it seems like the project is named `keyfinder-cli` rather than `keyfindercli`?